static/module.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<!--To test ECMA modules, run `npm run build` and then `npm start`
+    in the root KaTeX directory and then visit with a web browser
+    which supports modules: http://localhost:7936/module.html
+  -->
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>KaTeX Test</title>
+        <link rel="stylesheet" type="text/css" href="/dist/katex.css">
+        <link rel="stylesheet" type="text/css" href="/main.css">
+        <script type="module" type="text/javascript">
+            import katex from '/dist/katex.mjs';
+
+            const input = document.getElementById("input");
+            const math = document.getElementById("math");
+
+            input.addEventListener("input", reprocess, false);
+            reprocess();
+
+            function reprocess() {
+                try {
+                    katex.render(input.value, math, {displayMode: true, throwOnError: false, macros: {}});
+                } catch (e) {
+                    if (e.__proto__ === katex.ParseError.prototype) {
```

Comment:
I know we do some funky stuff in `ParseError`, does it prevent `e instanceof katex.ParseError` from working?

Reply:
@kevinbarabash No, I don't think so. I just copied this from `static/index.html`.